tests/unit/exception_logging_test.cc

```diff
@@ -200,9 +200,11 @@ BOOST_AUTO_TEST_CASE(throw_with_backtrace_exception_logging) {
         log_msg << std::current_exception();
     }
 
+#ifdef Seastar_HAVE_EXECINFO
```

Comment:
In the long run, this ifdef is wrong: Eventually we'll want to implement a backtrace correctly even if HAVE_EXECINFO is false, using some other method.
One way you can solve this is to do in backtrace.hh something like:
```c++
#if __has_include(<execinfo.h>)
#include <execinfo.h>
#define HAVE_EXECINFO
#endif
...
backtrace() {
#ifdef HAVE_EXECINFO
... some real definition
#else
   // Not implemented yet
#define SEASTAR_BACKTRACE_UNIMPLEMENTED
#endif
```

Then in the test you can check for `#ifndef SEASTAR_BACKTRACE_UNIMPLEMENTED`. In the future, we can implement it, and remove this macro, even when execinfo.h doesn't exist.

Reply:
Yes that is simpler